doc/SONiC-Configuration-Set-HLD.md

```diff
@@ -0,0 +1,67 @@
+# Terminology
+CS: Configuration SET
+
+# Summary
+This design propose an secure and light weight service that will provide a transparent and automated configuration query/subscription on given device set.
+
+# Scope
+This design only works on devices having gNMI server equipped.
+
+# Config DB Extention
+For example if we want to query/subscribe aggregate address on given device set:
+```json
+{
+    ...
+    "CONFIGURATION_SET" : {
+        "BGP_AGGREGATE_ADDRESS": {
+            "set_members" : [ "ip1", "ip2", "ip3", "ip4" ], # It contains all devices' ip address in the same set
+            "query_per_second" : "n", # it means it will query every peer n times per second
```

Comment:
will user input 0.1 if they want to query members every 10s? 
is query_interval in ms better?